README.md

```diff
@@ -59,7 +59,18 @@ Note how the cucumber-tsflow Decorators are being used to bind the methods in th
 
 ###### Compiling your TypeScript Support Code
 
-You'll also need a `tsconfig.json` file to compile your code. You'll also need to ensure that the `"moduleResolution": "node"` compiler option is set in order to bring in the typings that are shipped with cucumber-tsflow.
+You'll also need a `tsconfig.json` file to compile your code. The minimum requirements are:
```

Comment:
How about we just mention that `"experimentalDecorators": true` is required?

If we include the module and target options then it suggests that they have to be fixed at those values.

Reply:
Hi @timjroberts and @swaz90

Does it mean we can't use `import` ?

I am struggling with running cucumberJs test with @open-wc which throw an error 

```
import chai from './import-wrappers/chai.js';
       ^^^^

SyntaxError: Unexpected identifier
```

Just wondering if it's related topic.